Sources/Card/Models/Card.swift

```diff
@@ -1,32 +1,60 @@
 import Foundation
 
-/// Initialize a card object
-/// - Parameters:
-///   - number: The full card number, or PAN.
-///   - expirationMonth: The card's expiration month in MM format.
-///   - expirationYear: The card's expiration year in YYYY format.
-///   - securityCode: The card's security code (CVV, CVC, CVN, CVE, or CID).
 public struct Card: Encodable {
 
-    public var number: String
-    public var expirationMonth: String
-    public var expirationYear: String
-    public var securityCode: String
-
     enum CodingKeys: String, CodingKey {
         case number
         case expiry
         case securityCode
+        case cardHolderName = "name"
+        case billingAddress
     }
 
+    /// The primary account number (PAN) for the payment card.
+    public var number: String
+
+    /// The card expiration month
+    public var expirationMonth: String
+
+    /// The card expiration year
+    public var expirationYear: String
+
+    /// The three- or four-digit security code of the card. Also known as the CVV, CVC, CVN, CVE, or CID.
+    public var securityCode: String
```

Comment:
Looks like Android made this optional, can we do the same?

Reply:
I asked this in api-unification-team channel once before: https://paypal.slack.com/archives/C01UHMGB4DS/p1632412932043000
And for our use case this should always be required :smile: